conjure-java-jersey-server/build.gradle

```diff
@@ -1,14 +1,18 @@
 apply plugin: 'com.palantir.external-publish-jar'
-apply plugin: 'com.palantir.revapi'
 apply plugin: 'com.palantir.metric-schema'
 
+versionsLock {
+    disableJavaPluginDefaults()
+}
+
 dependencies {
     api "com.palantir.conjure.java.api:errors"
-    api "org.glassfish.jersey.core:jersey-server"
+    api "org.glassfish.jersey.core:jersey-server:2.26"
```

Comment:
The previous jersey version was 2.22.2, this bumps to 2.26 because I was having significant issues with version alignment otherwise.